spec/test-data/translations/glotpress-download/expected/values-fr/strings.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Translation-Revision-Date: 2021-03-31 09:54:04+0000
+Plural-Forms: nplurals=2; plural=n > 1;
+Generator: GlotPress/2.4.0-alpha
+Language: fr
+-->
+<resources>
+    <string name="product_attributes_error_saving">Erreur lors de l’enregistrement de vos attributs</string>
+    <string name="shipping_label_payments_cant_edit_warning">Seul le propriétaire du site peut gérer les moyens de paiement des étiquettes d’expédition. Veuillez contacter le propriétaire de la boutique %1$s (%2$s) pour gérer les moyens de paiement.</string>
+    <string name="shipping_label_shipping_method_free_shipping">Livraison gratuite</string>
+    <string name="shipping_label_shipping_method_local_pickup">Point de vente</string>
+    <string name="shipping_label_shipping_method_flat_rate">Forfait</string>
+    <string name="shipping_label_shipping_carrier_flat_fee_banner_message">Le client a payé %1$s (%2$s) pour la livraison</string>
+    <string name="shipping_label_woo_discount_bottomsheet_message" formatted="false">Lors de l’achat d’étiquettes d’expédition avec WooCommerce, vous bénéficiez de 5 à 40 %% de réduction par rapport aux tarifs de la Poste.</string>
```

Comment:
Contains `%%` on purpose (while `formatted="false"`) to test that we generate a warning

Reply:
Would it make sense to document this and the other choices as a comments in the file? before this line?

```suggestion
    <!-- Contains %% with formatted="false" on purpose to test that we generate a warning -->
    <string name="shipping_label_woo_discount_bottomsheet_message" formatted="false">Lors de l’achat d’étiquettes d’expédition avec WooCommerce, vous bénéficiez de 5 à 40 %% de réduction par rapport aux tarifs de la Poste.</string>
```

Not sure what the implications would be with the localization process.

Reply:
No it wouldn't because those files are the fixtures and are used in the unit test expectations to be compared at character level with the actual result we get as an output. And that output we get when running the action doesn't have those comments in the input stub…

So we can't add comments to the fixtures without also making the character-level comparison test fail.

BUT that's also why I've also written unit tests that ensure that those fixtures still contain cases like that (iow if we modify the fixtures in the future and accidentally remove that case of %% that we put there intentionally to cover that case, the unit test that I added to specifically test that we do have at least one fixture with %% + formatted=false… will fail, to remind us that we need to have a fixture covering this  case.

Reply:
In other words, this is "documented" by the test on lines 139-144 of the helper spec file.

Reply:
Gotcha. Makes sense 👍